src/include/rho/FixedVector.hpp

```diff
@@ -222,9 +222,8 @@ namespace rho {
 	// Virtual function of GCNode:
 	void detachReferents() override;
     private:
-	T* m_data;  // pointer to the vector's data block.
```

Comment:
@kmillar In which cases would the data be placed elsewhere?

I'm running a benchmark of this commit right now. By the way, if the data is stored remotely then both `m_data` and `m_first_element_storage` are redundant so maybe we could just reuse one of them as the pointer anyway?